user_guide_src/source/changelogs/v4.3.3.rst

```diff
@@ -28,6 +28,9 @@ Changes
 Deprecations
 ************
 
+- **Text Helper:** :php:func:`random_string()`'s type ``basic``, ``md5``, and
```

Comment:
```suggestion
- **Text Helper:** :php:func:`random_string()`'s type ``basic``, ``md5`` and
```

Reply:
I am in favor of keeping the [Oxford Comma](https://www.grammarbook.com/blog/commas/oxford-comma/) 😊

Reply:
As always something new to learn, great link. 
Thanks